internal/nodes/avi_model_advl4_translator.go

```diff
@@ -202,7 +202,9 @@ func (o *AviObjectGraph) ConstructSvcApiL4VsNode(gatewayName, namespace, key str
 		}
 
 		// configures VS and VsVip nodes using infraSetting object (via CRD).
-		buildL4InfraSetting(key, avi_vs_meta, vsVipNode, nil, &gw.Spec.GatewayClassName)
+		if infraSetting := getL4InfraSetting(key, nil, &gw.Spec.GatewayClassName); infraSetting != nil {
```

Comment:
Previously there was no check for whether infraSetting is nil or not, now if it's nil - does it behave like older behaviour?

Reply:
yes, same behaviour.